frontend/src/api/index.ts

```diff
@@ -45,3 +64,37 @@ export const fetchCourse = async (
     return null;
   }
 };
+
+export const searchCourses = async (
+  searchQuery: string,
+  minIndex: number = 0,
+  maxIndex: number = 10
```

Comment:
Just curious what these query params are used for, do they just specify what range of results to return like for pagination?

Reply:
yup, it's just the range of results search will give back to us